tests/bpf2c_tests/expected/atomic_instruction_others_sys.c

```diff
@@ -286,7 +286,7 @@ static void
 _get_version(_Out_ bpf2c_version_t* version)
 {
     version->major = 0;
-    version->minor = 17;
+    version->minor = 18;
```

Comment:
it would make this PR easier to review if eBPF version changes were performed as standalone PRs.

Reply:
I see this will be resolved by #3690